docs/_docs/configuration/sass.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Sass/SCSS Options
+permalink: "/docs/configuration/sass/"
+---
+
+By default, Jekyll looks for Sass partials in the `_sass` directory relative to your site's `source` directory. You can change the default load path or specify additional load paths with the following options:
+
+* **sass_dir** - A filesystem-path which should be searched for Sass partials.
+* **load_paths** - An array of additional filesystem-paths which should be searched for Sass partials.
+
+<div class="note info">
+  <p>
+    Note that filesystem-paths specified in the <code>sass_dir</code> and
+    <code>load_paths</code> options are resolved relative to your site's
+    <code>source</code>, not relative to the location of the
+    <code>_config.yml</code> file.
+  </p>
+</div>
+
+The following additional Sass configuration options are offered by the [jekyll-sass-converter](https://github.com/jekyll/jekyll-sass-converter) plugin.
+
+* **style** - Sets the style of the CSS-output. Can be `nested`, `compact`, `compressed`, or `expanded`. See the [Sass docs](https://sass-lang.com/documentation/cli/dart-sass#style) for details.
```

Comment:
Recommended markdown style is dashes for outer level and stars for one level down. And 4 spaces for bullet points rather than 2 (for readability and because 4 spaces handles things better like code blocks within bullet points.

Also `CSS output` instead of `CSS-output`.
e.g.

```md
- **style** - Sets the style of the CSS output. Can be `nested`, `compact`, `compressed`, or `expanded`. See the [Sass docs](https://sass-lang.com/documentation/cli/dart-sass#style) for details.
- **sourcemap** - Controls when source maps shall be generated.
    * `never` — causes no source maps to be generated at all.
```

Also, since you split `sourcemap` values on multiple lines, you could do that here.

```md
- **style** - Set the style of the CSS output. See the [Sass docs](https://sass-lang.com/documentation/cli/dart-sass#style) for details. Valid values:
    * `nested`
    * `compact`
    * `compressed`
    *  `expanded`.
```